### DISCUSSION TITLE
Type error: Cannot Find Module

### DISCUSSION THREAD

**Discussion by roadtouk** [calcom/docker#281] [General]
While trying to build my own image running into this error time and again. 

`@calcom/web:build: Type error: Cannot find module '../../../../tests/libs/__mocks__/app-store' or its corresponding type declarations.`

And the build fails.

How can I solve this?

**Comment by krumware** [comment]
This has been reported as a big in the cal.com repo as well
Please upvote there as well
https://github.com/calcom/cal.com/issues/11217
